src/main/java/com/puppycrawl/tools/checkstyle/checks/indentation/LineWrappingHandler.java

```diff
@@ -39,6 +39,36 @@
 public class LineWrappingHandler {
 
     /**
+     * Enum to be used for test if first line's indentation should be checked or not.
+     */
+    public enum IgnoreFirstLine {
+        /**
+         * First line's indentation should NOT be checked.
+         */
+        TRUE,
```

Comment:
@romani I am currently thinking along a similar pattern for my other recommendation.
The name of the enum should be `LineWrappingOptions` and the values should be `IGNORE_FIRST_LINE` and `NONE`.

> Use
> Ignore

I feel this is so similar to the current `TRUE`/`FALSE` that it won't give us much room to add a 3rd option if we require it in the future.